webpack.config.js

```diff
@@ -11,7 +11,7 @@ module.exports = {
   plugins: [
     new CopyPlugin([
       { from: 'manifest.konnector' },
-      { from: 'package.json' },
```

Comment:
You need to keep package.json file. It is needed by the registry. At least I previouly go error from the registry when the version in package.json was not the same as the tag @CPatchane ?

Reply:
It's required by the registry indeed.